salt/environment_settings.yml

```diff
@@ -716,6 +716,13 @@ environments:
         - fluentd
         - rabbitmq
       rds:
+        - name: mitxpro
+          multi_az: True
+          engine: postgres
+          vault_plugin: postgresql-database-plugin
+          db_instance_class: db.t2.medium
+          allocated_storage: 100
+          public_acces: False
```

Comment:
This will require public_access to be True in order for Heroku to communicate with it